warrior/Framework/Utils/data_Utils.py

```diff
@@ -160,7 +161,17 @@ def get_credentials(datafile, system_name, myInfo=[], tag_name="system",
         output_dict = {}
         if len(myInfo) == 0:
             for child in element:
```

Comment:
per war-1165  the datype support should be implemented for both get_credentials and getSystemData .. I dont see it implemented for getSystemData

Reply:
data type support in the below code addition is implemented only if myInfo is empty it should be implemented for te case when users requests specific tags by providing specific values in myInfo.

Reply:
Sure Sathya, will work on this. Can you please provide any cases with non-empty myInfo for my testing?

Reply:
@navvrnd 
for non empty myInfo you can refer to Actions/DemoActions/demo_actions.py testset_calibration keyword line 142-143 
            credentials = get_credentials(self.datafile, call_system_name, \
                                ['calibration', 'user', 'location', 'testdata'])

Reply:
done. Added test case for checking myInfo and getSystemData from the demo actions as well